provision/infinispan/ispn-helm/values.yaml

```diff
@@ -3,7 +3,7 @@
 # Declare variables to be passed into your templates.
 
 replicas: 3
-defaultImage: quay.io/infinispan/server:15.0.4.Final
+defaultImage: quay.io/infinispan-test/server:PR-12480
```

Comment:
As agreed in the meeting, we're Ok to merge with a test image so that we can test the changes in this PR sooner rather than later. Instead of using `quay.io/infinispan-test/server:PR-12480` I can update the PR to use `quay.io/infinispan-test/server:15.0.x` which is affectively a preview image of `15.0.5.Final`

Reply:
That is interesting, is it some nightly image of Inifnispan? Or how often the 15.0.x is built?

Reply:
I this it could make sense to use it by default, we can discuss it, I added it as an item to the CrossDC board.